staging/src/k8s.io/kms/SECURITY_CONTACTS

```diff
@@ -0,0 +1,15 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+aramase
```

Comment:
AFAIK, `sig-auth-encryption-at-rest-approvers` is only relevant in `k/k` because the alias is defined here. If this alias is repo specific, we need to re define the alias in the staging dir, so the staging github repo has it or else the staging github repo will reference an alias that needs to be looked up in `k/k`.